src/components/dashboard/index.js

```diff
@@ -123,13 +128,21 @@ const Dashboard = (props) => {
         );
     }
 
+    // The base ID for the dashboard.
+    const baseId = fns.makeID(ids.ROOT);
+
     return (
-        <div
-            ref={dashboardEl}
-            id={fns.makeID(ids.ROOT)}
-            className={classes.gridRoot}
-        >
+        <div ref={dashboardEl} id={baseId} className={classes.gridRoot}>
             {isLoading ? <DashboardSkeleton /> : componentContent}
+            {detailsApp !== null && (
+                <AppDetailsDrawer
+                    appId={detailsApp?.id}
+                    systemId={detailsApp?.system_id}
```

Comment:
This could be shorter (and cover more falsey cases):
```suggestion
            {detailsApp && (
                <AppDetailsDrawer
                    appId={detailsApp.id}
                    systemId={detailsApp.system_id}
```
Also no need to check for falsey `detailsApp` again when getting its IDs.